src/tools/cephfs_mirror/PeerReplayer.cc

```diff
@@ -1611,7 +1611,7 @@ void PeerReplayer::peer_status(Formatter *f) {
       f->dump_string("state", "idle");
     } else {
       f->dump_string("state", "syncing");
-      f->open_object_section("current_sycning_snap");
```

Comment:
oooh.. didn't some tests fail due to this?

Reply:
No, the same mistake is in the test also. So it succeeds. I've fixed the typo in this PR just to avoid invalid search results in editors, which would be annoying.

Reply:
I see, only one typo in the test source. good catch!